pyulog/params.py

```diff
@@ -105,3 +101,30 @@ def main():
                 output_file.write(str(values[0]))
 
             output_file.write('\n')
+
+    elif args.format == "qgc":
+
+        for param_key in param_keys:
+            sys_id = 1
```

Comment:
You could get those from the parameters, but I'm not sure if it's worth it.

Reply:
Yeah I looked but wasn't sure how to do it.